sysdig/resource_sysdig_secure_organization.go

```diff
@@ -0,0 +1,154 @@
+package sysdig
+
+import (
+	"context"
+	v2 "github.com/draios/terraform-provider-sysdig/sysdig/internal/client/v2"
+	cloudauth "github.com/draios/terraform-provider-sysdig/sysdig/internal/client/v2/cloudauth/go"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"time"
+)
+
+func resourceSysdigSecureOrganization() *schema.Resource {
+	timeout := 5 * time.Minute
+
+	return &schema.Resource{
+		CreateContext: resourceSysdigSecureOrganizationCreate,
+		DeleteContext: resourceSysdigSecureOrganizationDelete,
+		ReadContext:   resourceSysdigSecureOrganizationRead,
+		UpdateContext: resourceSysdigSecureOrganizationUpdate,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(timeout),
+			Update: schema.DefaultTimeout(timeout),
+			Read:   schema.DefaultTimeout(timeout),
+			Delete: schema.DefaultTimeout(timeout),
+		},
+		Schema: map[string]*schema.Schema{
```

Comment:
Based on our offline discussion today, shall we change this in the schema to have the below aligning with the POST `/organizations` API -
- cloud_provider_type (required)
- managementAccountId (required)
- organizationalUnitIds (required)
- id (computed)

We won't need customer_id, organization_id, cloud_provider_id here no?

Reply:
Let's revisit this once haresh completes the /organizations refactor. Will get more idea, like managementAccountId and  organizationalUnitIds are AWS remnants might not need for gcp

Reply:
fixed in the latest push.